### ISSUE TITLE
quick question

### ISSUE THREAD

**Issue by cesalo** [dimitrijp/Organizr-Nginx#4]
First of all thanks a lot for putting this together.
I don't have a lot of experience using nginx and ssl but I got my different services running on ssl and nginx like sonarr, radarr .. etc .. working on urls like this: https://www.domain.com/sonnar or https://www.domain.com/radarr etc.
Now I'm having issues configuring organizr with ssl. I could configure organizr without ssl: http://www.domain.com:8183 .. I have it running in port 8183.
My question is using your scripts .. what port is Organizr running? 80? .. How could I adjust your conf. to use port 8183?
Thanks in advance.

**Comment by dimitrijp** [comment]
hi...sory for late reply..I dont quite understand why would you use 8183 port for your web server. By default or my config files, organizer is accessible via port 80 witch is redirected to 443 for SSL.

**Comment by cesalo** [comment]
Thanks for you reply. The main reason is because I'm using docker for all my different services/apps even organizr. Organizr docker is running using port 8183 in the host. I was finally able to use it using: fastcgi_pass 127.0.0.1:8183; with your examples and a lot of trial and error.
Thanks a lot of putting this together. It helped me a lot considering I just learned about this when I opened this issue.
